NU394-S2024TTh/savor
# Demo2

### Description

Please explain the changes you made here.

### QA

Please explain in detailed steps how the reviewer can test your changes and what the expected behaivors are (visual would help!). Consider different test cases/scenarios.

- User can upload images and changes will be reflected in DB (test database for the moment)
- User can sign in and same data is reflected across devices 

#### NOTE: this deleted some legacy fb code - need to make sure this stays consistent (code does not interract kindly w/ fb code)

### Packages

Please describe any additional packages you added to package.json.

### Linear Issue Tracking

Fixes SMA-47

### Checklist
